aten/src/ATen/CPUApplyUtils.h

```diff
@@ -151,6 +151,7 @@ struct strided_tensor_iter_fixed {
   int64_t strides_[N] = {0};
 
   strided_tensor_iter_fixed(strided_tensor_iter_fixed const&) = delete;
+  bool Overflow() const { return CanOverflow; };
```

Comment:
This is the nittiest of nits, but could we please use a method name that matches the prevailing method naming style (namely, lower case method)?

Reply:
While we're on the topic, let's have a comment? :)